repo/packages/K/kong/1/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packagingVersion": "3.0",
+  "name": "kong",
+  "version": "0.10.3",
+  "scm": "https://github.com/Mashape/kong",
+  "maintainer": "https://getkong.org/community/",
+  "website": "https://getkong.org/",
+  "description": "Kong is open-source API Gateway and Microservices Management Layer, delivering high performance and reliability.",
+  "framework": false,
+  "preInstallNotes": "You need to have Kong supported DB and Marathon-LB instances running",
```

Comment:
Add the link to dcos/example at the end of `preInstallNotes`. 
https://github.com/dcos/examples/tree/master/kong